test/functional/p2p_getaddr_caching.py

```diff
@@ -70,11 +71,8 @@ def run_test(self):
         cur_mock_time = int(time.time())
         for i in range(N):
             addr_receiver_local = self.nodes[0].add_p2p_connection(AddrReceiver())
-            addr_receiver_local.send_and_ping(msg_getaddr())
```

Comment:
There is a similar pattern below on lines 108,110,112. Should be possible to remove those too? If yes, then also `from test_framework.messages import msg_getaddr`.

To other reviewers: the `GETADDR` message is sent at the end of `P2PInterface::on_version()`.

Reply:
yes, I missed those, now I removed them all!